site/using-weave/weave-cloud.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Using Weave Cloud to get started with Weave Net
+menu_order: 10
+---
+
+Weave Net provides a simple to deploy networking solution for containerized apps. Here, we describe how to manage a Weave container network using a sample application which consists of two simple `netcat` services deployed to containers on two separate hosts.
+
+This section contains the following topics:
+
+ * [Signing in to Weave Cloud](#signing-in)
+ * [Launching Weave Net](#launching)
+ * [Launching on Additional Hosts](#peer-connections)
+ * [How Peer Discovery Works](#how-it-works)
+
+### <a name="signing-in"></a>Signing in to Weave Cloud
+
+Weave Cloud is a fast and simple way to visualize, manage and monitor
+containers and microservices.  In order to use Weave Cloud to set up
+Weave Net, you must sign up to Weave Cloud and create an "Instance".
+
```

Comment:
The point of this PR is the former - instead of typing in other hosts to connect to, you use Weave Cloud.
There already is some visualisation, which we can improve over time.